psi4/share/cmake/psi4/psi4Config.cmake.in

```diff
@@ -74,7 +74,7 @@ set(psi4_FOUND TRUE)
 set(PSI4_FOUND TRUE)
 
 function(add_psi4_plugin TARGET SOURCES)
-    # remove ${TARGET} from ${ARGV} to use ${ARGV} as ${SOURCES}
```

Comment:
Yeah, that could probably be done. The compilers would need to be defined here as well.